app/models/station.rb

```diff
@@ -2,4 +2,33 @@ class Station < ActiveRecord::Base
   validates_presence_of :name, :dock_count, :installation_date, :city_id
   belongs_to :cities
 
+  def self.average_dock_count
+    Station.average("dock_count").round
+  end
+
+  def self.most_bikes
+    Station.maximum("dock_count")
+  end
+
+  def self.stations_with_most_bikes
+    Station.where(dock_count: Station.most_bikes).pluck("name")
+  end
+
+  def self.least_bikes
+    Station.minimum("dock_count")
+  end
+
+  def self.stations_with_least_bikes
+    Station.where(dock_count: Station.least_bikes).pluck("name")
+  end
+
+  def self.newest_stations
```

Comment:
So I'm guessing for this method to work, the installation_date will have to be a date data type vs a string. Issue I'm running into is that the string that it comes in won't coerce into a date type automatically(unless converted I'm assuming). Do you think that conversion should happen in the seed?